.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
Again, as an observation, I would be really, really unhappy if I was asked to have 8.5/10 on pylint in order to be able to contribute to mbed-os. A lot of pylint rules are pseudo-arbitrary at best. I believe it's a really bad idea to add this restriction.